.github/dependabot.yml

```diff
@@ -6,7 +6,7 @@ updates:
   # Python product dependencies
   #
 
```

Comment:
do we need to add a constraints file for tests as well?

Reply:
Good point, added.